### PR TITLE
Include kernel upgrade and reboot of compute nodes

### PR DESCRIPTION
**Pull request by dlundquist**
This combines the QEMU 2.0 backport from the Ubuntu cloud archive and kernel upgrade into to a single maintenance window.

### CODE REVIEW

File: playbooks/CVE-2015-3456.yml [hunk 1]
```diff
@@ -1,25 +1,72 @@
...
-  handlers:
-  - name: restart qemu-kvm
-    service: name=qemu-kvm state=restarted
+- hosts: compute:!db
+  tasks:
+  - name: Reboot
+    command: shutdown -r now
+    async: 0
+    poll: 0
+    ignore_errors: true
```
Comment (j2sol) [inline playbooks/CVE-2015-3456.yml]: failed_when: false is the better option, saves red output. But doesn't matter for this task.

**Comment by j2sol** [comment]
These changes look good.

**Comment by dlundquist** [comment]
Update qemu-kvm packages has been released into Ubuntu cloud archive: http://ubuntu-cloud.archive.canonical.com/ubuntu/pool/main/q/qemu/

**Comment by j2sol** [comment]
Closing this, the branch this is targeting is a closed PR as well.

**Pull request closed without merging**